uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/components/leave-request-popup-details-tab.component.spec.js

```diff
@@ -800,6 +800,10 @@ define([
               expect(controller.loading.showBalanceChange).toBe(false);
             });
 
+            it('does not recalculate the balance', function () {
```

Comment:
Please add some more definite unit tests like
1.  When Unit is in hour, it re calculates,
2. Otherwise it does not.